aws/scripts/sh/website_installation.sh

```
@@ -1,8 +1,12 @@
 echo #### Install Software
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)